sourcemash/frontend/static/js/views/sidenav_view.js

```diff
@@ -61,13 +71,18 @@ Sourcemash.Views.SidenavView = Backbone.View.extend({
     });
   },
 
+  refreshPage: function(e){
+    e.preventDefault();
+    window.location.reload(true);
```

Comment:
Let's just fetch/reset the categories again instead of reloading the whole page.